source/scripting_v3/GTA/Entities/Peds/NavMeshRouteResult.cs

```diff
@@ -0,0 +1,32 @@
+//
+// Copyright (C) 2015 crosire & contributors
+// License: https://github.com/crosire/scripthookvdotnet#license
+//
+
+using System;
+
+namespace GTA
+{
+	/// <summary>
+	/// Set of enumerations of the state of any active <see cref="TaskInvoker.FollowNavMeshTo(Math.Vector3, PedMoveBlendRatio, int, float, FollowNavMeshFlags, float, float, float, float)"/> task running on the <see cref="Ped"/>.
+	/// </summary>
+	public enum NavMeshRouteResult
+	{
+		/// <summary>
+		/// No navmesh task was found on the <see cref="Ped"/>.
+		/// </summary>
+		TaskNotFound,
+		/// <summary>
+		/// The task has not yet looked for a route.
+		/// </summary>
+		RouteNotYetTried,
+		/// <summary>
+		/// The task has tried & failed to find a route (will keep trying).
```

Comment:
Since the documentaiton text is XML, this has to be `&amp;`. `&` won't show correctly.
Also don't need to add `using System;` to files that only define an enum.